lib/matplotlib/backends/backend_svg.py

```diff
@@ -391,16 +391,19 @@ def _write_hatches(self):
                 x="0", y="0", width=str(HATCH_SIZE+1),
                 height=str(HATCH_SIZE+1),
                 fill=fill)
-            writer.element(
-                'path',
-                d=path_data,
-                style=generate_css({
+            hatch_style = {
                     'fill': rgb2hex(stroke),
                     'stroke': rgb2hex(stroke),
                     'stroke-width': str(mpl.rcParams['hatch.linewidth']),
                     'stroke-linecap': 'butt',
                     'stroke-linejoin': 'miter'
-                    })
+                    }
+            if stroke[3] < 1.:
```

Comment:
The `.` is unnecessary.

Reply:
Didn't know what your policy is regarding making float operations explicit. Will fix.